homeassistant/components/motioneye/__init__.py

```diff
@@ -231,3 +344,44 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         await config_data[CONF_CLIENT].async_client_close()
 
     return unload_ok
+
+
+class MotionEyeView(HomeAssistantView):
```

Comment:
Why not use the `webhooks` integration that Home Assistant has available, instead of writing your own solution?